docs/openapi/components/schemas/common/FSMACreatingCTE.yml

```diff
@@ -0,0 +1,125 @@
+$linkedData:
+  term: FSMACreatingCTE
+  '@id': https://w3id.org/traceability#FSMACreatingCTE
+title: FSMA Creating CTE
+description: >-
+  "Information regarding an FSMA Creating CTE (Critical Tracking Event), designed to meet reporting
+  requirements for the FDA Traceability Rule as specified here:
+  https://www.fda.gov/food/food-safety-modernization-act-fsma/fsma-proposed-rule-food-traceability"
+type: object
+properties:
+  type:
+    oneOf:
+      - type: array
+        items:
+          type: string
+          enum:
+            - FSMACreatingCTE
+      - type: string
+        const: FSMACreatingCTE
+  foodCreated:
+    title: Food Created
+    description: The traceability food that was created.
+    $ref: ./FSMAProduct.yml
+    $linkedData:
+      term: foodCreated
+      '@id': https://w3id.org/traceability#FSMAProduct
+  locationCreated:
+    title: Location Created
+    description: Where the product was created.
+    $ref: ./Place.yml
+    $linkedData:
+      term: locationCreated
+      '@id': https://schema.org/Place
+  dateCompleted:
+    title: Date Completed
+    description: The date creation was completed in 24hr ISO 8601 format (e.g. 1970-01-01T00:00Z for UTC, or 1970:01:01T00:00-5:00 for New York on standard time).
+    type: string
+    $linkedData:
+      term: dateCompleted
+      '@id': https://schema.org/DateTime
+  additionalData:
+    title: Additional Data
+    description: Additional Key Data Elements (KDEs).
+    type: array
+    items:
+      $ref: ./FSMAAbstractKDE.yml
+    $linkedData:
+      term: additionalData
+      '@id': https://w3id.org/traceability#FSMAAbstractKDE
+additionalProperties: false
+example: |-
+  {
+    "type": "FSMACreatingCTE",
+    "foodCreated": {
+      "type": "FSMAProduct",
+      "traceabilityLot": {
+        "type": "FSMATraceabilityLot",
+        "lotCode": "2b1ed51f-f1da-49f6-8130-ac81d1e5e61d",
+        "lotCodeAssignmentMethod": "Lot codes are generated as a random UUID.",
+        "lotType": "created"
+      },
+      "quantityUnitMeasure": {
+        "type": [
+          "QuantitativeValue"
+        ],
+        "unitCode": "Crates",
```

Comment:
sadly... not a unit code... https://schema.org/unitCode

Reply:
I think we have this same problem on several schemas.

Reply:
In this case FDA uses Unit code this way - we should say `FSMAUnitCode` or something like that @rhofvendahl

Reply:
How big is the coding system? We can define them all here possibly.
Or point to an existing ontology that defines them...

No hits in https://www.gs1.org/voc/

Have you checked https://qudt.org/ ?

Reply:
@OR13 the units are a little fuzzy, e.g. "crates", "pallets", "field bins".

@mprorock  correct me if I'm wrong but I think while standardized units for weight or volume might be useful, they're not currently part of the suggested reporting workflow (I'm guessing because users might not have easy access to that info &/or reviewers might not find that info particularly useful).

Looks like `https://schema.org/unitText` provides an alternative for units that don't have a standard unit code, which seems great for our purposes. Instead of referring to a QuantitativeValue I'll just provide `quantity` and `unit` properties where unit has id `unitText`.